scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -35,7 +35,8 @@ workflow GvsJointVariantCalling {
       Int SNP_VQSR_max_gaussians_override = 6
       Int SNP_VQSR_mem_gb_override = ""
     }
-    File gatk_override = "gs://gvs_quickstart_storage/jars/gatk-package-4.2.0.0-613-g1219576-SNAPSHOT-local.jar"
+    # This is the most updated snapshot of the code as of Oct 18, 2022
+    File gatk_override = "gs://gvs_quickstart_storage/jars/gatk-package-4.2.0.0-625-g950122c-SNAPSHOT-local.jar"
```

Comment:
we should probably update the jarmaking script to work the date in there but that's for another ticket / PR

Reply:
Agreed.  I added in this comment with the date precisely because I noticed the date wasn't present in the jar, but it won't be necessary once we update that jar script to add it into the filename